### PR TITLE
Aether 23 kafka

### PR DESCRIPTION
**Pull request by djmoran64**
This one is ready to go, fixed the test failures.  Updated the docker

### CODE REVIEW

File: .travis.yml
Comment (admbtlr) [inline .travis.yml]: Just remove these lines

File: kafka/data-src/AetherDB.json
Comment (admbtlr) [inline kafka/data-src/AetherDB.json:6]: Should this be `aether` instead of `gather2`? Can you check this against Douno's renaming PR pls?

**Review by admbtlr — CHANGES_REQUESTED** [review]

**Review by quesebifurcan — APPROVED** [review]
Tried running it locally without running into any issues 👍

File: .travis.yml
Comment (djmoran64) [reply .travis.yml]: Done.

File: kafka/data-src/AetherDB.json
Comment (djmoran64) [reply kafka/data-src/AetherDB.json:6]: It should, but if I change here it will break this branch and the tests may not pass.  I didn't know if that really matters or if I should wait until after douno’s merge.

**Comment by djmoran64** [comment]
With the Issues that encountered with Kafka, the team decided to hold off on this merge for now.  We will use logstash in the same way that we did for the demo and commit that after the big gather -> aether merge

**Pull request closed without merging**
